BATCH_MANUAL.md

```diff
@@ -0,0 +1,93 @@
+
+## How to use
+
+To utilize the batch script, follow these steps to configure LDAR-Sim with all parameters in a designated directory. Then, employ the following command line structure:
+```ldar_sim_batchrun.py --n_rep {number_of_repetitions} -P {directory_path}```
+
+Example command to run 100 batch simulations contained in the folder simulations would be:
+```ldar_sim_batchrun.py --n_rep 100 -P simulations```
+
+## Outputs
+
+### Regular Outputs
+
+The following files are maintained for each of programs of the first batch of simulations
+```timeseries_output_{simulation}_{Program name}.csv```
+```leaks_output_{simulation}_{Program name}.csv```
+
+The following file will be kept for the very last batch of simulations that are run.
+```sites_output_{simulation}_{Program name}.csv```
+
+### Sites
+
+`sites_output_{program}_concat.csv`
+
+|Column Names      |Unit   |Description                |
+|------------------|-------|---------------------------|
+|facility_ID       |N/A    |Facility ID of the site|
+|Cum_leaks         |N/A    |Average cumulative number of leaks at the site|
+|init_leaks        |N/A    |Average number of initial leaks found at the site|
+|total_emissions_kg|kg     |Average total emissions per site (in kilograms)|
+|subtype_code      |N/A    |The subtype code of the site|
+|lat               |Degrees|Latitude of the site|
+|lon               |Degrees|Longitude of the site|
+|equipment_groups  |N/A    |Number of equipment groups at the site|
+|n_sims            |N/A    |Number of simulations the site participated in|
+
+```sites_summary.csv```
+
+|Column Names                       |Unit       |Description                |
+|-----------------------------------|-----------|---------------------------|
+|Program                            | N/A       |Name of the program             |
+|Mean_emissions_per_site            |per site   |Mean emissions per site in kg|
+|5th_percentile_Emissions_per_site  |per site   |5th percentile of emissions per site in kilograms|
+|95th_percentile_Emissions_per_site |per site   |95th percentile of emissions per site in kilograms|
+|Mean_leaks_per_site                | # per site|Mean number of leaks per site|
+|5th_percentile_leaks_per_site      | # per site|5th percentile of the number of leaks per site at a given site|
+|95th_percentile_leaks_per_site     | # per site|95th percentile of the number of leaks per site at a given site|
+
+### Leaks
+
+```leaks_summary.csv```
+
+|Column Names                |Unit   |Description                |
+|----------------------------|-------|---------------------------|
+|Program                     | N/A   |Name of the program|
+|Volume_mean                 |kg     |Mean emissions volume in kilograms of leaks in the simulation|
```

Comment:
Clarity: As above, I think being extra clear about what these stats are is important. If we need to describe how they are calculated to clear on what these numbers are I think that's fine